examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -38,10 +38,12 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
+		19B5D85DA1BD905553A8E437 /* Pods-Runner.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-Runner.release.xcconfig"; path = "Pods/Target Support Files/Pods-Runner/Pods-Runner.release.xcconfig"; sourceTree = "<group>"; };
```

Comment:
You probably don't want this XCode project change in this PR: they're unrelated changes that are easily separable, so I'd put them in separate PRs. (or just revert this change if you didn't mean to make it).